### ISSUE TITLE
suit label

### ISSUE THREAD

**Issue by fainalo4** [marcypotter16/Cirulla#2]
Ciao, 
sto guardando al tuo lavoro per un mio progetto e ho notato una cosa.
Forse in **ConsolePlayer.py** riga 103 dovrebbe essere "Q" per i denari invece che "C"
Grazie

**Comment by marcypotter16** [comment]
Ciao,
Hai ragione, era un pezzo di codice che poi non ho usato per la versione grafica del gioco, grazie mille. Per curiosità il progetto a cui stai lavorando è questo https://rlcard.org/ ?

**Comment by fainalo4** [comment]
Esatto, sto adattando parte del tuo codice per modellare e inserire Cirulla tra i giochi supportati dal repository. Il sogno è creare un agente che si alleni a giocare "perfettamente" a Cirulla. Per ora sto lavorando qui: (https://github.com/fainalo4/rlcard-cirulla) Grazie per il tuo supporto :)
